release/release_logs/1.10.0/microbenchmark.json

```diff
@@ -0,0 +1,138 @@
+{
+  "single_client_get_calls": [
+    24730.95712317265,
+    239.22426527698045
+  ],
+  "single_client_put_calls": [
+    45715.93513986856,
+    366.82602393524735
+  ],
+  "multi_client_put_calls": [
+    189007.7322366993,
+    747.0416719411013
+  ],
+  "single_client_get_calls_Plasma_Store": [
+    5822.640574716487,
+    139.29057092881047
+  ],
+  "single_client_put_calls_Plasma_Store": [
+    5806.698088308273,
+    92.88490958691352
+  ],
+  "multi_client_put_calls_Plasma_Store": [
+    8755.183433422579,
+    198.96653021166597
+  ],
+  "single_client_put_gigabytes": [
+    20.274106255029984,
+    5.77023316809178
+  ],
+  "single_client_tasks_and_get_batch": [
+    15.216350139554642,
+    0.20511158958607925
+  ],
+  "multi_client_put_gigabytes": [
+    38.468680557353714,
+    1.9726230328400056
+  ],
+  "single_client_get_object_containing_10k_refs": [
+    13.524832217726612,
+    0.192034247128874
+  ],
+  "single_client_tasks_sync": [
+    1710.196782060312,
+    40.086361754373776
+  ],
+  "single_client_tasks_async": [
+    14390.477923818155,
+    271.36072682863403
+  ],
+  "multi_client_tasks_async": [
+    39730.8671520313,
+    2266.6052838083147
+  ],
+  "1_1_actor_calls_sync": [
+    2589.157291231204,
+    124.0599787184886
+  ],
+  "1_1_actor_calls_async": [
+    6488.679319380362,
+    239.08621203545292
+  ],
+  "1_1_actor_calls_concurrent": [
+    5369.625993528198,
+    120.17942920626632
+  ],
+  "1_n_actor_calls_async": [
+    11741.041010762125,
```

Comment:
This seems to be slower: it was `15614.656699877709` in 1.9

Reply:
Adding more details: in `1.10.0` prior to the last round of cherry-picks, this number was `10247.98729565172`.
Int `1.10.0rc0`, it was `10403.857830375695`.

Reply:
Does this number mean the max number of calls that can be finished within unit time? @jjyao @architkulkarni

Reply:
This means the number of calls per second (throughput) so the higher the better.